pwndbg/gdblib/kernel/kallsyms.py

```diff
@@ -387,15 +397,139 @@ def parse_symbol_table(self):
             self.kallsyms[name[1:]] = (addr, name[0])
 
     def get_token_table(self):
-        tokens = []
-        position = self.token_table
+        if not self.is_uncompressed:
```

Comment:
Random thought: Is there any documentation how is this stored in kernel I could look into?

cc: @fidgetingbits @chrf01

Reply:
No documentation, only source code.
- [https://codebrowser.dev/linux/linux/kernel/kallsyms.c.html](https://codebrowser.dev/linux/linux/kernel/kallsyms.c.html)
- [https://github.com/marin-m/vmlinux-to-elf/blob/master/vmlinux_to_elf/kallsyms_finder.py
](https://github.com/marin-m/vmlinux-to-elf/blob/master/vmlinux_to_elf/kallsyms_finder.py)